ui/src/main/java/org/pentaho/mantle/client/workspace/JsJob.java

```diff
@@ -125,7 +137,11 @@ public final String getOutputPath() {
     String token = "outputFile = ";
     int index = resource.indexOf( token );
     if ( index < 0 ) {
-      return "";
+      token = "output file=";
```

Comment:
Should this match line 137 above "outputFile = " for consistency?  This string is looked for specifically elsewhere, so it shouldn't vary.

Reply:
I see in some cases its giving the string as following examples, so I needed to use both the tokens to see and take the output path.
input file = /home/admin/footrans.ktr:output file=/home/admin/footrans*
input file = /home/admin/footrans.ktr:outputFile = /home/admin/footrans-puc-6.*

Reply:
If this is the case, then there would be other bugs awaiting to be fixed. In this same file, other places test for `:outputFile = /`.

References to `output file=`: https://github.com/search?q=org%3Apentaho+%22output+file%3D%22&type=code

References to `:outputFile =`: https://github.com/search?q=org%3Apentaho+%22%3AoutputFile+%3D%22&type=code

This does not seem right. Should a spike be created to investigate?

Reply:
> If this is the case, then there would be other bugs awaiting to be fixed. In this same file, other places test for `:outputFile = /`.
> 
> References to `output file=`: https://github.com/search?q=org%3Apentaho+%22output+file%3D%22&type=code
> 
> References to `:outputFile =`: https://github.com/search?q=org%3Apentaho+%22%3AoutputFile+%3D%22&type=code
> 
> Possibly related new issue: https://hv-eng.atlassian.net/browse/BACKLOG-39834.
> 
> This does not seem right. Should a spike be created to investigate?

The code is been updated in a way that it dont look for the reference like either "output file=: " or "outputFile =:" or any other, it takes the output path which is been separated with : after the input path, and then it takes content after the "=" part and trim, so it gets correct path.